### PR TITLE
Export statement documentation.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Review by Marenz — DISMISSED** [review]
LGTM

File: test/libsolidity/ASTJSON/documentation_triple.sol [hunk 1]
```diff
@@ -0,0 +1,15 @@
+contract C {
+    /// test
+    uint a;
+    function f() public pure returns (uint x) {
+        /// test2
+        for (uint i = 0; i < 20; i++) {
+            /// test3
+            x *= 2;
+        }
+        /// test4
```
Comment (axic) [inline test/libsolidity/ASTJSON/documentation_triple.sol]: Can you add a natspec comment with spaces and quotation marks to ensure it is stored properly (escaped if needed)?

**Review by mijovic — COMMENTED** [review]
Looks good, but would be nice to cover with tests that Alex suggested.

File: test/libsolidity/ASTJSON/documentation_on_statements.sol [hunk 2]
```diff
@@ -0,0 +1,15 @@
+contract C {
+    // test
+    uint a;
+    function f() public pure returns (uint x) {
+        // test2
+        for (uint i = 0; i < 20; i++) {
+            // te st"3"
```
Comment (chriseth) [inline test/libsolidity/ASTJSON/documentation_on_statements.sol]: Ah ok, so those are not exported!

**Comment by chriseth** [comment]
@axic https://github.com/ethereum/solidity/pull/10924/files#diff-36f30032086ed03df171b5dde4599ea4205896fa663af0855f7a77c115cb00f5R54

**Review by mijovic — APPROVED** [review]

File: test/libsolidity/ASTJSON/documentation_triple.sol [hunk 3]
```diff
@@ -0,0 +1,16 @@
+contract C {
```
Comment (axic) [inline test/libsolidity/ASTJSON/documentation_triple.sol:1]: Sorry, probably should test for comments on contract too. Maybe `@spec` would have contract level invariants defined here?

And likewise can you add free functions, interfaces, and libraries?

**Pull request merged**